htdocs/core/class/html.form.class.php

```diff
@@ -7312,9 +7313,10 @@ public function selectDate($set_time = '', $prefix = 're', $h = 0, $m = 0, $empt
 						}
 						// Note: We don't need monthNames, monthNamesShort, dayNames, dayNamesShort, dayNamesMin, they are set globally on datepicker component in lib_head.js.php
 						if (!getDolGlobalString('MAIN_POPUP_CALENDAR_ON_FOCUS')) {
+							$buttonImage = $calendarpicto ?: DOL_MAIN_URL_ROOT . "/theme/" . dol_escape_js($conf->theme) . "/img/object_calendarday.png";
```

Comment:
Using DOL_MAIN_URL_ROOT here will generate pb: the url used may differe when using the compononent from backoffice.
Why not keeping DOL_URL_ROOT ? If we need to use the component from another context like a website, we can use the new parameter. Did i miss something ?

Reply:
Yes, using the personalized icon will allow the component to be utilized in an external context. I updated the PR and reverted to using DOL_URL_ROOT